src/pages/wishlist/wishlist.module.css

```diff
@@ -0,0 +1,23 @@
+.like-btn {
+  top: 5%;
+}
+.wishlist-card {
+  height: 26.2rem;
+}
```

Comment:
26.2 if it is calculated than it should be fine else it should not be used as it would be difficult to scale it later.

Reply:
its intentional